pom.xml

```diff
@@ -29,7 +29,7 @@
     </scm>
 
     <properties>
-        <clojure.version>1.3.0</clojure.version>
```

Comment:
`nrepl` uses require+refer which was apparently missing at that time :) maybe we can specify minimum clj version or setup a travis matrix for all supported versions